src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,417 @@
-Customer Response
-=================
-
-We deliberately use the term **Customer Response** as an amalgamation of the
-terms **Customer Support** and **Incident Response** to represent the fact that
-both activities are involved here. **Customer Support** involves responding to
-inquiries about the *use* of VisIt_. *How do I do X? Y is not working for me,*
-*can you help? I am having trouble getting VisIt_ to work in client/server*.
-These are examples of typical **Customer Support** inquiries VisIt_ developers
-are routinely enlisted to answer.
-
-In some cases, user's inquiries reveal a deeper, systemic, recently occurring
-*operational* issue which may involve either an issue with the VisIt_ software
-itself (e.g. a botched VisIt_ installation, an incompatability in client/server
-protocol, a missing database reader plugin, etc.) or an issue with underlying
-computational infrastructure upon which VisIt_ depends (incompatible grahics
-drivers, a downed file system or network, a security environment trigger).
-These latter issues are typical of **Incident Response** services and require
-rapid response in diagnosing and enlisting resources to correct.
-
-As is typical of most DOE software project teams, VisIt_ developers wind up
-having to serve in both the roles of **Customer Support** and
-**Incident Response**. So, we call this activity **Customer Response**.
-While some aspects of **Customer Response** are under the direct control of
-VisIt_ developers, many aspects are not and involve collaborations with other
-teams in resolving. In most cases the extent of the VisIt_ team's involvement in
-*operations* is confined primarly to the VisIt_ software itself; its
-development, testing, release and deployment which includes installations we
-directly support on systems where we have accounts, hosted binary downloads for
-common platforms and the tools to build from sources.
+Software Reliability Engineering
```

Comment:
You've noticed my schizophrenia here 😄 . For a long time, I had trouble taking the ref @griffin28 sent and incorporating that nomenclature here because of the use of the word `Site` instead of `Software` in `Site Reliability Engineering`.  Once I decided it might be ok to take some liberties and call it `Software Reliability Engineering`, then that whole phrase vibed with me more *and*, more importantly, connected our activites more closely with commercial sector (which I think our sponsors always like to see because it suggests we're *staying current* with modern practices). But, since `Customer Response` by that time was all over this document, I opted for a short-cut, just re-titling the doc and nothing else. Long story short, my thinking has evolved since originally drafting.

Does the above vibe with anyone else?

If so, maybe it makes sense for me to de-bold/cap all **Customer Response** refs here and, where appropriate, replace with *SRE*. So, instead of this being a **Customer Response** process its an **SRE** process.